broken-line/contracts/LibBrokenLine.sol

```diff
@@ -31,7 +37,15 @@ library LibBrokenLine {
     using SignedSafeMathUpgradeable for int;
     using SafeMathUpgradeable for uint;
 
-    function add(BrokenLineDomain.BrokenLine storage brokenLine, BrokenLineDomain.Line memory line, uint cliff) internal {
+    /**
+     *add Line, save data in LineData
+     **/
+    function add(BrokenLineDomain.BrokenLine storage brokenLine, uint id, BrokenLineDomain.Line memory line, uint cliff) internal {
+        require(line.slope != 0, "Slope == 0, unacceptable value for slope");
+        require(line.slope <= line.bias, "Slope > bias, unacceptable value for slope");
+        require(brokenLine.initiatedLines[id].line.start == 0, "Line with given id is already exist");
```

Comment:
I think it's better to check line's bias. it should not be 0. start can be 0

Reply:
ok, agree